### PR TITLE
Fix: v1.4.0-rc1 feedback issues & bugs

### PR DESCRIPTION
**Pull request by sillydan1**
## v1.4.0-rc1 feedback
This PR solves the following problems: 
 - issue #55 in 6ae4327
 - issue #57 in ccae65c
 - sometimes location x/y coords are loaded completely wrong (fixed in c65d973)
 - multiline tags will occationally have a vscrollbar (fixed in 1d73c1e)

### CODE REVIEW

File: src/main/java/dk/cs/aau/huppaal/presentations/TagPresentation.java [hunk 1]
```diff
@@ -101,13 +108,12 @@ private void initializeLabel() {
...
-            newHeight += resY;
+            newWidth += resX + (GRID_SIZE * 0.1);
+            newHeight += resY + (GRID_SIZE * 0.1);
 
-            text.setMinWidth(newWidth);
             text.setPrefWidth(newWidth);
-            text.setMinHeight(TAG_HEIGHT);
-            text.setPrefHeight(TAG_HEIGHT);
+            text.setPrefHeight(newHeight);
+            //System.out.printf("%f x %f (%f x %f)%n", newWidth, newHeight, newBounds.getWidth(), newBounds.getHeight());
```
Comment (Nielswps) [inline src/main/java/dk/cs/aau/huppaal/presentations/TagPresentation.java]: Remove before merge

**Review by Nielswps — DISMISSED** [review]
Interesting that the shift check is not required, but I guess it should be pretty general for all applications

File: src/main/java/dk/cs/aau/huppaal/presentations/TagPresentation.java (hunk 1 above)
Comment (sillydan1) [reply src/main/java/dk/cs/aau/huppaal/presentations/TagPresentation.java]: Fixed in 085db9d

**Comment by sillydan1** [comment]
> Interesting that the shift check is not required, but I guess it should be pretty general for all applications

Yea 🤔 I was surprised too. I guess that if you need some more custom behavior, you'd just have to work around this behavior...

**Pull request merged**